### ISSUE TITLE
Pre-training logs

### ISSUE THREAD

**Issue by eric-haibin-lin** [google-research/bert#279]
Hi,

Did anyone try to run pre-training using the scripts provided? Can anyone share the pre-train log?

**Comment by 2009wp2012** [comment]
what's the problem you encountered ？

**Comment by mjc14** [comment]
@eric-haibin-lin did you reproduce  pre-training  the result? i  mod  version of pytorch bert code to reproduce result, i am nearly  successful.  i can get the same  result  about  next  sentence and mask lm prediction. but it can not get same result for some nlp tasks fine-tune. my result is 3 point less than official base Chinese model.

**Comment by eric-haibin-lin** [comment]
@mjc14 No I haven't. Running full pre-training is very expensive and I am wondering if anyone wants to share their pre-training result. What dataset did you train on? What is the next sentence and mask lm result you get? Which fine-tune task were you using?
I hope to pre-train from a very different dataset from scratch and want to know what the training loss look like if done from scratch

**Comment by mjc14** [comment]
@eric-haibin-lin  i  use Simplified Chinese wiki2018(1G)。 next sentence accuracy is  97~98%，mask lm  is 77~80%。  my fine-tune task is NER  and it is my dataset。 i think my dataset  is less than  official dataset。
